site/en/docs/extensions/migrating/publish-mv3/index.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: 'layouts/doc-post.njk'
+title: Publish your extension
+seoTitle: Publishing a new Manifest V3 Chrome extension.
+description: 'Guidance for publishing a new Manifest V3 extension'
+date: 2023-09-04
+# updated: 2023-05-10
+---
+
+After [converting your extension][migration-checklist] to Manifest Version 3, it's time to release it on the [Chrome Web Store][cws]. There are a few ways to publish your new release gradually. This article discusses releasing a beta version and slowly increasing the number of testers, gradually rolling out to your user base, and monitoring your extension review status to quickly publish any fixes if needed.
+
+## Publish a beta testing version {: #publish-beta }
+
+The Chrome Web Store provides a way to publish to beta testers. This way, a subset of users can report any issues before publishing to the rest of your user base. 
```

Comment:
I'd be good to add one sentence explaining why this might be helpful during the mgiration.